great_expectations/compatibility/bigquery.py

```diff
@@ -18,24 +16,89 @@
 except (ImportError, AttributeError):
     sqlalchemy_bigquery = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
 
-BIGQUERY_TYPES: Dict[str, Any] = (
-    {
-        "INTEGER": sqlalchemy_bigquery.INTEGER,
-        "NUMERIC": sqlalchemy_bigquery.NUMERIC,
-        "STRING": sqlalchemy_bigquery.STRING,
-        "BIGNUMERIC": sqlalchemy_bigquery.BIGNUMERIC,
-        "BYTES": sqlalchemy_bigquery.BYTES,
-        "BOOL": sqlalchemy_bigquery.BOOL,
-        "BOOLEAN": sqlalchemy_bigquery.BOOLEAN,
-        "TIMESTAMP": sqlalchemy_bigquery.TIMESTAMP,
-        "TIME": sqlalchemy_bigquery.TIME,
-        "FLOAT": sqlalchemy_bigquery.FLOAT,
-        "DATE": sqlalchemy_bigquery.DATE,
-        "DATETIME": sqlalchemy_bigquery.DATETIME,
-    }
-    if sqlalchemy_bigquery
-    else {}
-)
+try:
+    from sqlalchemy_bigquery import INTEGER
+except (ImportError, AttributeError):
+    INTEGER = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+try:
+    from sqlalchemy_bigquery import NUMERIC
+except (ImportError, AttributeError):
+    NUMERIC = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+try:
+    from sqlalchemy_bigquery import STRING
+except (ImportError, AttributeError):
+    STRING = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+try:
+    from sqlalchemy_bigquery import BIGNUMERIC
+except (ImportError, AttributeError):
+    BIGNUMERIC = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+try:
+    from sqlalchemy_bigquery import BYTES
+except (ImportError, AttributeError):
+    BYTES = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import BOOL
+except (ImportError, AttributeError):
+    BOOL = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import BOOLEAN
+except (ImportError, AttributeError):
+    BOOLEAN = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import TIMESTAMP
+except (ImportError, AttributeError):
+    TIMESTAMP = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import TIME
+except (ImportError, AttributeError):
+    TIME = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+try:
+    from sqlalchemy_bigquery import FLOAT
+except (ImportError, AttributeError):
+    FLOAT = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import DATE
+except (ImportError, AttributeError):
+    DATE = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+try:
+    from sqlalchemy_bigquery import DATETIME
+except (ImportError, AttributeError):
+    DATETIME = SQLALCHEMY_BIGQUERY_NOT_IMPORTED
+
+
+class BIGQUERY_TYPES:
+    """Namespace for Bigquery dialect types"""
+
+    INTEGER = INTEGER
+    NUMERIC = NUMERIC
+    STRING = STRING
+    BIGNUMERIC = BIGNUMERIC
+    BYTES = BYTES
+    BOOL = BOOL
+    BOOLEAN = BOOLEAN
+    TIMESTAMP = TIMESTAMP
+    TIME = TIME
+    FLOAT = FLOAT
+    DATE = DATE
+    DATETIME = DATETIME
```

Comment:
Here I'm expanding on a pattern we have already established for Snowflake/Databricks/Postgres. You will see below I move the dictionary to another file that has similar dictionaries.